crates/aptos-crypto/src/ed25519.rs

```diff
@@ -468,7 +463,10 @@ impl TryFrom<&[u8]> for Ed25519Signature {
     type Error = CryptoMaterialError;
 
     fn try_from(bytes: &[u8]) -> std::result::Result<Ed25519Signature, CryptoMaterialError> {
-        Ed25519Signature::check_malleability(bytes)?;
+        // We leave this check here to detect mauled signatures earlier, since it does not hurt
+        // performance much. (This check is performed again in Ed25519Signature::verify_arbitrary_msg
+        // and in ed25519-dalek's verify_strict API.)
+        Ed25519Signature::check_s_malleability(bytes)?;
```

Comment:
iiuc, this slows down deserializations but I guess we don't change anything in this pr